packages/webapp/next.config.mjs

```diff
@@ -14,9 +14,14 @@ const nextConfig = {
     {
       source: "/dao/proposals",
       destination: "/dao",
-      permanent: false,
+      permanent: false
     },
-  ],
+    {
+      source: "/dao/proposals/:id", // redirect any 3rd party legacy proposal links to layer 1
+      destination: "/dao/proposals/eth/:id",
+      permanent: false
```

Comment:
no reason to not make this a permanent redirect imo

Reply:
lol, I'll try it out. SInce the `/dao/proposals` was already `false` I was going to change both of these redirects to be `true`, but something wasn't working that might have had nothing to do with this config, so to save time I just let it be. But I'll give it another try